common/tools/dev-tool/src/commands/samples/prep.ts

```diff
@@ -73,14 +92,7 @@ async function* cat<T>(...generators: AsyncIterable<T>[]): AsyncIterable<T> {
 }
 
 export default leafCommand(commandInfo, async (options) => {
-  let argumentDir;
```

Comment:
@willmtemple -- I've converted this to using `--directory` instead of `args[0]` ... A quick look at instances where `dev-tool samples prep` is used didn't reveal any usages that would be broken by this change.

Reply:
Changes to this script all LGTM.